### PR TITLE
Added tenure filters | psp-4804

### PR DESCRIPTION
**Pull request by FuriousLlama**
image

### CODE REVIEW

**Comment by devinleighsmith** [comment]
Hmm, are we not unit testing any of this functionality?

**Comment by devinleighsmith** [comment]
highway/road details filter does not appear to work.

**Review by devinleighsmith — CHANGES_REQUESTED** [review]
highway/road details filter doesn't seem to do anything.

File: source/frontend/src/components/maps/leaflet/Control/AdvancedFilter/FilterContentForm.tsx [hunk 1]
```diff
@@ -49,10 +49,28 @@ export const FilterContentForm: React.FC<React.PropsWithChildren<IFilterContentF
   }, [initialFilter, onChange]);
 
   const { getByType } = useLookupCodeHelpers();
+
+  // Propery options
```
Comment (asanchezr) [inline source/frontend/src/components/maps/leaflet/Control/AdvancedFilter/FilterContentForm.tsx:53]: typo: Propery -> Property

**Comment by FuriousLlama** [comment]
> Hmm, are we not unit testing any of this functionality?

The filter is already being tested. I could add a test for creating the snap, but I am not sure how much value that would bring

**Pull request merged**